src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -16,6 +16,9 @@
 
 package org.springframework.cloud.deployer.spi.kubernetes;
 
+import io.fabric8.kubernetes.api.model.NodeAffinity;
```

Comment:
it would be preferred not to be exposing fabric8 types, rather create wrappers similar to `Tolerations`, `PodSecurityContext` in this class

Reply:
But this is a complex type which requires several sub-types. Do you want me to just mimic it cloning the same structure?

Reply:
ideally they would be wrapped. understood about the graph size. i merged the PR as-is and will revisit that issue at a later time. thanks!